### DISCUSSION TITLE
hope to show beatmapsets of all modes in lazer

### DISCUSSION THREAD

**Discussion by alisy-Qnh** [ppy/osu#23039] [Ideas] [answered]
in the current version of lazer, it seems that the beatmapsets of all modes cannot be displayed. when you select a mode, the beatmapsets of other modes will be hidden.

this is convenient most of the time, but it bothers me when I want to listen to music or play casually. it would be nice if a future version could add the ability to display all beatmapsets.

**Comment by Theighlin** [comment, marked as answer]
#6500
